### PR TITLE
Update enrollment-restrictions-set.md

### PR DESCRIPTION
**Pull request by nahira-ms**
Add iOS is supporeted IMEI.

Add blocking personal device on macOS

### CODE REVIEW

**Comment by PRMerger9** [comment]
@nahira-ms : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/enrollment/enrollment-restrictions-set.md [hunk 1]
```diff
@@ -94,12 +94,18 @@ If you block Android Enterprise work profile enrollment on personal devices, onl
 
 #### Blocking personal iOS/iPadOS devices
 By default, Intune classifies iOS/iPadOS devices as personally-owned. To be classified as corporate-owned, an iOS/iPadOS device must fulfill one of the following conditions:
-- [Registered with a serial number](corporate-identifiers-add.md).
+- [Registered with a serial number or IMEI](corporate-identifiers-add.md).
 - Enrolled by using Automated Device Enrollment (formerly Device Enrollment Program)
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/enrollment-restrictions-set.md:98]:
```suggestion
- Enrolled by using Automated Device Enrollment (formerly Device Enrollment Program).
```
```suggestion
- Enrolled by using Automated Device Enrollment (formerly Device Enrollment Program)
```

File: memdocs/intune/enrollment/enrollment-restrictions-set.md [hunk 2]
```diff
@@ -94,12 +94,18 @@ If you block Android Enterprise work profile enrollment on personal devices, onl
...
 
 > [!NOTE]
 > An iOS User Enrollment profile overrides an enrollment restriction policy. For more information, see [Set up iOS/iPadOS and iPadOS User Enrollment (preview)](ios-user-enrollment.md).  
 
+#### Blocking personal macOS devices
+By default, Intune classifies macOS devices as personally-owned. To be classified as corporate-owned, an macOS device must fulfill one of the following conditions:
+- [Registered with a serial number](corporate-identifiers-add.md).
+- Enrolled by using Automated Device Enrollment (formerly Device Enrollment Program)
+
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/enrollment-restrictions-set.md:107]:
```suggestion
- Enrolled by using Automated Device Enrollment (formerly Device Enrollment Program).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @nahira-ms, kindly commit the minor suggested changes. Many thanks.

**Comment by PRMerger5** [comment]
@lenewsad : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@lenewsad : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by lenewsad** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by lenewsad** [comment]
#sign-off

**Pull request merged**